Editor/CanvasView.cs

```diff
@@ -244,35 +245,55 @@ public void Load(Graph graph)
 
             // TODO: Move into reflection
             var attrs = graph.GetType().GetCustomAttributes(true);
+
+
             foreach (var attr in attrs)
             {
+                //Add Tags for search provider
                 if (attr is IncludeTagsAttribute include)
                 {
                     searchWindow.IncludeTags.AddRange(include.Tags);
                 }
+
+                //Add Required nodes from GraphAttributes
+                if (attr is RequireNodeAttribute required)
+                {
+                    if (searchWindow.IncludeTags.Contains(required.type.Namespace.ToString()))
+                    {
+                        Node node = graph.GetNode(required.type);
+                        if (node == null)
+                        {
+                            node = NodeReflection.Instantiate(required.type);
+                            node.Graph = graph;
+                            node.Name = required.nodeName;
+                            node.Position = required.position;
+                            AddNodeFromSearch(node, node.Position, null, false);

```

Comment:
This call will probably incorrectly place the node, since `AddNodeFromSearch` will transform the input position to something relative to the current view. But we don't have an equivalent `AddNode` to procedurally insert nodes onto the graph through the canvas. 

I'll merge this in for now as-is, but I'll make a change afterwards to add a separate `AddNode` method - that ultimately `AddNodeFromSearch` will call with an updated position.